compute_endpoint/tests/unit/test_endpointmanager_unit.py

```diff
@@ -2162,6 +2162,13 @@ def test_load_user_config_schema(
         assert "user config schema is not valid JSON" in str(a)
 
 
+@pytest.mark.parametrize("ext", (".yaml.j2", ".yaml"))
+def test_load_user_config_schema_extensions(conf_dir: pathlib.Path, ext: str):
+    template_path = conf_dir / f"user_config_template{ext}"
+    template_path.write_text('{"foo": "bar"}')
+    load_user_config_schema(conf_dir)
```

Comment:
- Consider adding an assertion on the return value.  From the myopic point of view of this test, the `load_user_config_schema` might as well be `lambda x: 1`.  The assertion cements in stone that it did _something_.
- Since the changelog mentions that one takes priority, consider adding a test that that is in fact the case.  (i.e., in the presence of both files, the correct one is loaded.)

Reply:
All set.